examples/travis.example.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - 4
+  - 5
```

Comment:
sometimes we also add 'stable'. Since Node.js will continue to grow and stable will always be the LTS, we should consider making it part of default.